shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -192,7 +194,9 @@ public boolean onGenericMotionEvent(@NonNull MotionEvent event) {
     boolean isMovementEvent =
         (event.getActionMasked() == MotionEvent.ACTION_HOVER_MOVE
             || event.getActionMasked() == MotionEvent.ACTION_SCROLL);
-    if (!isPointerEvent || !isMovementEvent) {
+    if (isPointerEvent && isMovementEvent) {
+      continue;
```

Comment:
Looks like the CI failure is here, continue outside of loop.

Reply:
yeah sorry This is still in draft.